src/d3a/models/state.py

```diff
@@ -55,10 +55,7 @@ def get_energy_production_forecast_kWh(self, time_slot, default_value=None):
         return self._energy_production_forecast_kWh[time_slot]
 
     def get_available_energy_kWh(self, time_slot, default_value=None):
-        if default_value is not None:
-            available_energy = self._available_energy_kWh.get(time_slot, default_value)
-        else:
-            available_energy = self._available_energy_kWh.get(time_slot)
+        available_energy = self._available_energy_kWh.get(time_slot, default_value)
```

Comment:
This can raise an error in the line below if it fell down to the default None value

Reply:
That's true, even though it doesn't depend on this specific PR if I'm not mistaken

Reply:
Fixed!